libtest

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env sh
-
-PYTHONPATH=python/src python -m unittest discover -s python/test/lib -p '*_libtest.py'
```

Comment:
Have swe moved this test somewhere? Is it still run in travis?

Reply:
File restored, thanks for catching this.